includes/breadcrumbs.inc

```diff
@@ -104,8 +104,9 @@ function islandora_solr_get_breadcrumbs_recursive($pid, array &$context = NULL)
 function islandora_solr_get_breadcrumb_parent($pid) {
   module_load_include('inc', 'islandora_solr', 'includes/query_processor');
   $solr_build = new IslandoraSolrQueryProcessor();
-  $solr_query = "PID:(" . islandora_solr_lesser_escape($pid) . " OR " .
-    islandora_solr_lesser_escape("info:fedora/" . $pid) . ")";
+  $solr_query = format_string('PID:"!pid"', array(
```

Comment:
This won't pass coder and the `!` replacement is no longer supported. Advise use of [strtr](http://php.net/manual/en/function.strtr.php).